.prettierignore

```diff
@@ -0,0 +1,5 @@
+# Ignore artifacts
+build
+
+# Ignore token file
+token.js
```

Comment:
```suggestion
tokens.js
```
If this file doesn't appear to do anything, let's just not use it, I think. We can probably just delete it.

Reply:
Are you talking about the token.js file?

Reply:
Hmm... I tried the .prettierignore, but maybe I didn't structure it properly. I'll give it another go and see if that works! Thank you for sharing that👍

Reply:
> If this file doesn't appear to do anything, let's just not use it, I think. We can probably just delete it.

Is this in reference to the `token.js` file?

Reply:
Sorry...i mean the prettier ignore. The prettierignore might also need to be in place before you make changes? Now that the file is already saved in a certain format, changing the prettierignore won't revert it back to it's previous formatting.

Reply:
For this one I think we can just delete `prettierignore` file, and then leave `tokens.js` as it currently stands in this PR. Not a huge deal.

Reply:
Oh! The `.prettierignore` file! Sorry, I think I got confused there. I thought you were giving a suggestion on how to make it work. I see now that you are referencing what is already there, but not functioning and suggesting that we take the `.prettierignore` file out. Is that right? Totally on board with that.

Reply:
yeah...let's just remove it. 👍